src/Simulation/Simulators/ResourcesEstimator/ResourcesEstimator.cs

```diff
@@ -78,7 +78,11 @@ public ResourcesEstimator(QCTraceSimulatorConfiguration config) : base(config)
             }
             else if (name == MetricsNames.WidthCounter.ExtraWidth)
             {
-                return "Width";
+                return "WidthLowerBound";
```

Comment:
As a brief note, this could break user code. At the moment, users may extract the `"Width"`  key from the returned dictionary. I think this even happens in samples, such that I'd suggest firing off a cross-repo to validate this API change.

Reply:
Yes, this _is_ a breaking change. How do we usually stage changes like this? Shall we provide both properties in parallel for a while and document that "Width/Depth" are deprecated?

Reply:
@bettinaheim could speak much better to .NET API deprecation policies; I'm mainly responsible for Q# API deprecation policies, and for maintaining samples / libraries. From that stand point, a cross-repo build could tell us if any current samples are broken and give us the heads-up we need to fix that. Otherwise, the cross-repo `master` would break when merging this in.